packages/core/src/components/Spinner/Spinner.scss

```diff
@@ -113,11 +113,22 @@ Style guide: components.spinner
 
 - If the process takes a long time, use something else.
 
+## Accessibility Considerations
+
+- Spinners are useful to announce a page load sequence, and should be paired with a parent container that includes several ARIA attributes:
+  * `aria-relevant="additions text"`
+  * `aria-live="polite"`
+  * `aria-atomic="false"`
+- aria-relevant may include additions, text, and removals. If a section is going to have items added or removed, use additions and removals. If there will be additions or text changes without explicit removals, use additions and text.
```

Comment:
Nit: Can we format these HTML attributes and values as code? `aria-relevant`, `additions`, `text`, `removals`, etc

Reply:
Sure thing. Should these not be list items, and just inline code, or maybe marked up as a solid block of HTML that displays all of the contextual information?